main.c

```diff
@@ -1,6 +1,106 @@
 #include <stdio.h>
+#include <stdlib.h>
 
+#define MAX_FOOD_NAME 100
+#define MAX_SPECIFIC_FOOD_NAME 100
+#define MAX_DRINK_NAME 100
+void readmenuitem(char *thing);
 int main() {
-    printf("Hello, World!\n");
+    int nofood;
+    printf("Please input number of food types: \n>");
+    scanf("%d", &nofood);
+    char** food=(char**)malloc(nofood * sizeof(char*));
+    printf("Please input food types (each on a new line, may contain spaces):\n");
+    for(int i=0;i<nofood;i++)
+    {
+        printf(">");
```

Comment:
you don't have to print this, the execution example was shown with ">" in order to differentiate between user input & output